src/templates/static/legal/tac.jsx

```diff
@@ -27,6 +29,11 @@ const Tac = () => (
                             { id: 'legal-mt',     text: it.L('MetaTrader') },
                         ],
                     },
+                    {
+                        id  : 'affiliate',
+                        text: it.L('Affiliate Programme', it.website_name),
```

Comment:
what is it.website_name supposed to replace?

Reply:
Oh geez. sorry for that.